src/main/kotlin/com/codingchallenge/transactionanalyser/TransactionLoader.kt

```diff
@@ -0,0 +1,35 @@
+package com.codingchallenge.transactionanalyser
+
+import com.codingchallenge.transactionanalyser.domain.Transaction
+import com.codingchallenge.transactionanalyser.domain.TransactionBuilder
+import java.io.BufferedReader
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.util.stream.Collectors
+
+class TransactionLoader(private val input: InputStream) {
+    fun loadTransactionsFromCsv(): List<Transaction>{
+        val transactions: List<Transaction>
+        BufferedReader(InputStreamReader(input)).use {
```

Comment:
have not seen us use `.use` in the team much. this is a great use case and I like the brevity.